package.json

```diff
@@ -58,7 +59,7 @@
     "babel-plugin-istanbul": "^6.1.1",
     "cypress": "^9.5.3",
     "dotenv": "^8.2.0",
-    "eslint": "^7.2.0",
+    "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-import": "^2.22.1",
```

Comment:
@Kesari3008,
Should we add engine details in package.json
```
"engines": {
        "node": ">=20.13"
    }
```
and refer node version in .nvmrc

`20`

Did you verify if in local development node version is selected as 20+ while running project ?

Reply:
@Neeraj-swarnkar Updated the engine details, .nvmrc file is already updated with the codename of node 20. I checked the node version using command node -v while running npm install and npm run build and in the circle ci pipeline is updated to use node 20.13. Is there any other way you would like me top verify if node 20+ is being used in local development.

Reply:
@Kesari3008, Thanks for making the changes,

For local development, Same command using  `node -v `
Also if nvm present `nvm ls ` will tell which one is currently active node version,
Switch to some other node version using nvm use and do yarn install or npm install and check nvm ls